src/raw.rs

```diff
@@ -279,7 +280,12 @@ where
                         // time to schedule it.
                         if state & RUNNING == 0 {
                             // Schedule the task.
-                            Self::schedule(ptr);
+                            Self::schedule(
+                                ptr,
+                                ScheduleInfo {
+                                    woken_while_running: false,
+                                },
```

Comment:
You repeat this a lot, maybe add a crate-private function to initialize it

Reply:
Added crate-private function `ScheduleInfo::new()` and applied to those repetitions.